inference-engine/thirdparty/clDNN/src/graph_optimizer/basic_memory_dependencies.cpp

```diff
@@ -36,6 +36,40 @@ void basic_memory_dependencies::run(program& p) {
             add_memory_dependency(it, node);
         }
 
+        if (node->is_type<convolution>()) {
```

Comment:
This is quite similar to the code in network.cpp. But slightly outdated. Maybe can we merge these two?

Reply:
I suppose now it's better to stay both cases to preserve current dependencies.